Maple2.Server.Game/Manager/ExperienceManager.cs

```diff
@@ -112,7 +112,10 @@ public sealed class ExperienceManager {
             return;
         }
 
-        long expValue = 0;
+        long expValue;
+        short charLevel = session.Player.Value.Character.Level;
+        int fieldLevel = session.Field.Metadata.Drop.Level;
+        int correctedLevel = fieldLevel > charLevel ? charLevel : fieldLevel;
```

Comment:
IF these are only used in a specific case in the switch, let's move it within there.
Also, `charLevel` is redundant. you can already get that from `Level` in the ExperienceManager.

Reply:
fixed